bin/plugin/commands/packages.js

```diff
@@ -371,6 +372,16 @@ async function publishPackagesToNpm( {
 	minimumVersionBump,
 	releaseType,
 } ) {
+	const { stdout: nodeVersion } = await command( 'node -v', {
+		cwd: gitWorkingDirectoryPath,
+	} );
+	log( `>> Current node version: ${ nodeVersion }.` );
```

Comment:
do we know if the currently-running script will run with the same `node` that we could call here? if so, the identical version string should be available at `process.version`

Reply:
My hope was to investigate whether we could run a different version of node and npm in the spawned processes, but apparently, nvm is architected in a way that doesn't allow that. We can as well, remove this logging as I will open an issue explaining the issue we have when running the script with Node 16/npm 8 while the branches from previous WP releases (`wp/X.Y`) enforce Node 14/npm 6 or even older versions.

Reply:
Removed with https://github.com/WordPress/gutenberg/pull/53565/commits/71462817fefaf319501635d23a374ce209a63991.